src/Security/test/Authentication.Mtls.Test/ClientCertificateAuthenticationTests.cs

```diff
@@ -590,7 +584,7 @@ public async Task VerifyValidationEventPrincipalIsPropagated()
 
         var server = new TestServer(builder)
         {
-            BaseAddress = baseAddress
+            BaseAddress = baseAddress!
```

Comment:
Why doesn't this crash at runtime? Should the parameter be declared as non-nullable? Or only set the BaseAddress property if the parameter is not `null`?

Reply:
I don't know, but I'm not sure it is worth spending any time here... A bunch of changes to this file slipped into this PR that shouldn't have, but the whole file gets removed in #1311 anyway